internal/repositories/repRegistry.go

```diff
@@ -0,0 +1,11 @@
+package repositories
+
+var usersDataRep usersDataRepository
+
+func RegisterUserRepository(rep *usersDataRepository){
```

Comment:
Honestly, I don't understand why do you need DataRepository and UserRepository

Reply:
lets discuss it

Reply:
ok, it was done only to try in repRegistry.go approach where we return Interface for working in controllers. I have no special reason for this))) The idea was that if we will create another implementation we will need only make changes in repRegistry GetUSerRepository meyhod